static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx

```diff
@@ -273,9 +291,17 @@ export default function EndpointOverview() {
                   {t('Performance Only')}
                 </SegmentedControl.Item>
               </SegmentedControl>
+              <Button
+                style={{position: 'absolute', right: 0}}
+                size="sm"
+                onClick={handleViewAllEventsClick}
+              >
+                View All
+              </Button>
             </SegmentedControlContainer>
             <IssuesTable
               issueCategory={issueFilter === 'ALL' ? undefined : issueFilter}
+              httpMethod={method as string}
```

Comment:
Why do you need to cast this?

Reply:
method comes from the url query, which means its of type `string | string[]`.
In practice it should always come back as a `string` so I just decided to case it. It might be more robust if we just checked for array instead, but I thought to defer this as this code changes quite frequently.